migrations/000004_version.up.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+SET search_path = public, pg_catalog;
+
+--
+-- A table of database versions along with the date they were applied.
+--
+CREATE TABLE IF NOT EXISTS version (
```

Comment:
Yeah, IMO we can remove this from the golang-migrate version. golang-migrate will just ignore it in the existing DB and it can hang around until such a time as we don't need it anymore (which might even just be "when we merge this DB into de-database", hopefully)

Reply:
I removed it. Should be fixed now.